lib/bloc/account/account_bloc.dart

```diff
@@ -105,6 +112,9 @@ class AccountBloc {
   Sink get optimizationWhitelistRequestSink =>
       _permissionsHandler.optimizationWhitelistRequestSink;
 
+  final _exchangeRateController = new BehaviorSubject<Rates>();
```

Comment:
We don't need this controller as we have List<FiatConversion> on the account model. Any consumer of exchange rate should be able to get it by listening to the account stream.